src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_TripSearchScreen/components/CityZoneMessage.tsx

```diff
@@ -40,39 +44,66 @@ export const CityZoneMessage: React.FC<CityZoneMessageProps> = ({
   const fromCityZone = useFindCityZoneInLocation(from, cityZones);
   const toCityZone = useFindCityZoneInLocation(to, cityZones);
 
-  if (!fromCityZone || !toCityZone) {
+  if (!fromCityZone?.enabled || !toCityZone?.enabled) {
```

Comment:
@gorandalum a Fix for https://github.com/AtB-AS/mittatb-app/pull/3576#issuecomment-1560844490 was made here.